test/integ_test/evaluator/docker_eval/dockerfile/Dockerfile

```diff
@@ -13,4 +13,4 @@ RUN pip install onnxruntime \
             onnxconverter_common \
             olive-ai==0.1.0
 
-WORKDIR /olive
+WORKDIR /olive
```

Comment:
I think these whitespaces will probably fail the formatting checks. 

Could you please install and run pre-commit? We have the hooks already defined. 
```
pip install -r requirements-dev.txt
pre-commit install 
# the first time, run on all files
pre-commit run --all-files
```

After the first time, pre-commit will automatically run on any changed files every time you git commit.

Reply:
Thanks for your help. The PR code is now checked with pre-commit